src/rest-server/src/templates/dockerContainerScript.mustache

```diff
@@ -177,18 +179,41 @@ function  azure_rdma_preparation()
 azure_rdma_preparation
 {{/ azRDMA }}
 {{/ reqAzRDMA }}
-# Write env to system-wide environment
-env | grep -E "^PAI|PATH|PREFIX|JAVA|HADOOP|NVIDIA|CUDA" > /etc/environment
+
+function write_user_command()
+{
+  local scriptPath="$1"
+
+  # Read an escaped string to an array and unescape
+  local lines=$'{{ taskData.command }}'
```

Comment:
what if there's single quotation mark `'` in command?

Reply:
It should come escaped from `{{ }}`.

Reply:
The `{{ }}` is used in mustache syntax, instead of bash, nodejs will replace the `{{ taskData.command }}` part with real command. [Here's](http://mustache.github.io/mustache.5.html) how it works.

For example, if the real command is `echo "Hello"`, nodejs will generate this line from dockerContainerScript.mustache to `local lines=$'echo "Hello"'` in dockerContainerScript.sh. The `{{ }}` won't be kept in runtime bash.